Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -361,3 +361,20 @@ open class MediaAttachment: NSTextAttachment {
         textContainer?.layoutManager?.invalidateLayout(for: self)
     }
 }
+
+
+// MARK: - NSCopying
+//
+extension MediaAttachment: NSCopying {
+
+    public func copy(with zone: NSZone? = nil) -> Any {
+        let clone = type(of: self).init(identifier: identifier, url: url)
+        clone.image = image
+        clone.extraAttributes = extraAttributes
```

Comment:
`extraAttributes` is copy on write. I think we're good with that, those values are immutable, to the best of my knowledge!